Dangerfile.swift

```diff
@@ -540,4 +541,3 @@ do {
 } catch {
     fail("Failed to find out the correct check routine. Please check if your PR is created from or into a correct branch.")
 }
```

Comment:
<!--
  0 failure: 
  1 warning:  Files should have...
  
  
  DangerID: danger-id-Danger;
  File: Dangerfile.swift;
  Line: 543;
-->

- :warning: Files should have a single trailing newline. (`trailing_newline`)